### ISSUE TITLE
Add Skip Link to Bypass Table

### ISSUE THREAD

**Issue by SamiyahKey** [cisagov/manage.get.gov#3144]
### Story

As a keyboard-only user,
I need a skip link that allows me to bypass the table in the main content area and jump directly to the filters,
So that I can efficiently navigate the page without unnecessary repetition.

URL: Domain Request (/admin/registrar/domainrequest/)

### Acceptance Criteria

- [x] Add a skip link at the top of the page and just above the table to allow users to bypass the table and navigate directly to the filters.
    Example: `<a href="#filters" class="skip-link" aria-label="Skip to the filters section">Skip to filters</a>`
- [x] The skip link should be visually hidden but accessible via keyboard and become visible when focused.
- [x]  Screen readers must announce the presence of the skip link using correct ARIA labels in programmatic markup, providing clear context for its function.
- [ ]  Confirm with a designer that the implementation adheres to WCAG 2.4.1 and 2.1.1

### Additional Context

- Ensure skip links are compatible with various assistive technologies.
- Test the implementation to confirm that the skip link is accessible in different browser and screen reader combinations.

**Comment by witha-c** [comment]
Removed "skipped design" to reflect that an update to design documentation or libraries is not needed.
